doc/rtd/reference/modules.rst

```diff
@@ -88,6 +88,9 @@ date. A 5 year timeline may also be expected for changed keys.
 
 .. automodule:: cloudinit.config.cc_users_groups
 .. automodule:: cloudinit.config.cc_wireguard
+
+.. _mod-write_files:
```

Comment:
Is this required? Shouldn't line 94: `.. automodule:: cloudinit.config.cc_write_files` suffice?

Reply:
This is so that it can be linked to. E.g., [Line 166](https://github.com/canonical/cloud-init/pull/5206/files#diff-684544d5084d7006625beb79d2dd05da94d91585222062d77197282c756b6968R166).

I'm not aware of a way of linking to an automodule otherwise, but if you know a way I can do that instead.